frontend/src/app/main/data/workspace/shortcuts.cljs

```diff
@@ -364,19 +364,12 @@
                           :subsections [:main-menu]
                           :fn #(st/emit! (dw/select-all))}
 
-   :toggle-grid          {:tooltip (ds/meta "'")
+   :toggle-guides          {:tooltip (ds/meta "'")
                           ;;https://github.com/ccampbell/mousetrap/issues/85
```

Comment:
Should this comment be aligned with the rest of the aatributes?
(it happens with more comments in this file)